### ISSUE TITLE
Homebrew formula is not up-to-date for 0.16.0 release

### ISSUE THREAD

**Issue by ceyhuno** [realm/SwiftLint#1197]

**Comment by aamctustwo** [comment]
This is a duplicate of #1173 `Homebrew hasn't been updated for v0.16.0`.

SwiftLint is still waiting on the Homebrew team to merge the PR https://github.com/Homebrew/homebrew-core/pull/8719
